torch/_torch_docs.py

```diff
@@ -3384,6 +3385,8 @@ def merge_dicts(*dicts):
     {input}
     {dim}
     {keepdim}
+    dtype (:class:`torch.dtype`, optional): the desired data type of returned tensor.
```

Comment:
this description isn't correct, we never use the default tensor type here.

Why do you need to specify the string in this example but not in the above one?